actors/runtime/exitcode/reserved.go

```diff
@@ -2,7 +2,7 @@ package exitcode
 
 import "strconv"
 
-type ExitCode int64
+type ExitCode uint8
```

Comment:
We discussed this at length in person in November and since:
- we only use two integer types, with no unsigned variants
- there's no reason to restrict actors error code ranges, and if we did we can do it at runtime

Please revert this for now and cast in your shim

Reply:
I don't recall these conversations, but we can make this change later, sure.

Reply:
Thanks!